localstack/constants.py

```diff
@@ -5,12 +5,6 @@
 # LocalStack version
 VERSION = localstack.__version__
 
-# constant to represent the "local" region, i.e., local machine
-REGION_LOCAL = "local"
-
-# dev environment
-ENV_DEV = "dev"
-
```

Comment:
The Environment logic was removed in https://github.com/localstack/localstack/pull/9644